ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -807,7 +807,29 @@ objects:
           cat >> /tmp/artifacts/installer/install-config.yaml << EOF
         proxy:
           httpsProxy: http://admin:admin@35.196.128.173:3128/
```

Comment:
are you intentionally keeping `httpsProxy` on HTTP?  I'd have expected you to use https:// for both `httpProxy` and `httpsProxy`.

Reply:
good catch! i wasn't...

Reply:
also, they're two different proxies... the non-tls is pointed to by `httpProxy` so that should stay `http`

Reply:
i had accidentally added the tls proxy info to the `httpProxy` field... fixed